src/emulator/shader/src/translator/alu.cpp

```diff
@@ -127,6 +127,7 @@ bool USSETranslatorVisitor::vmad(
     spv::Id vsrc2 = load(inst.opr.src2, write_mask, 0);
 
     if (vsrc0 == spv::NoResult || vsrc1 == spv::NoResult || vsrc2 == spv::NoResult) {
+        LOG_ERROR("Source not loaded (vsrc0: {}, vsr1: {}, vsrc2: {})", vsrc0, vsrc1, vsrc2);
```

Comment:
No point printing sources out. You can just do
```suggestion
        LOG_ERROR("Source not loaded!");
```

Same with others

Reply:
Also, can you go make sure that all the disasm are printed before this error? It' easier to debugging if that is correct

Reply:
no is needed, and velocityu have request me i remove this !, and here have 2 or 3 different src, so i have added this for can know quickely what exaclty is wrong

Reply:
Yeah, but don't print sources. It doesn't help anything for me though, and finding it in the SPIR-V code is useless.

If you insists to keep it,change them to src0 all the way (instead of vsrc0)

Reply:
hmm use "inst.opr.src0" get to much, and use just vsrc0 works good also, see on log few time so is good like that

Reply:
for one exemple just see here on other function
>[shader::usse::USSETranslatorVisitor::vtst]: Source not loaded (lhs: 0, rhs: 71)

so it said here lhs problem with 0, and rhs with 71

Reply:
ok